web-server/v0.4/src/layouts/BasicLayout.js

```diff
@@ -224,7 +224,7 @@ class BasicLayout extends React.PureComponent {
           // you will be forced to jump to the 403 interface without permission
           Authorized={Authorized}
           menuData={getMenuData()}
-          collapsed={collapsed !== true}
+          collapsed={collapsed}
```

Comment:
Can we consider reverting the default collapsed state to already collapsed? This allows for more space to view tables and data throughout the platform.

Reply:
@gurbirkalsi testing this out I can find that `collapsed` variable is misleading and passing `!collapsed` actually sets default state to collapsed,  unlike the other one

Reply:
Wait a minute, Let me test it out again though

Reply:
@gurbirkalsi You're right, my bad!